rpcserver_test.go

```diff
@@ -91,9 +92,9 @@ func testGetBlockHash(r *rpctest.Harness, t *testing.T) {
 }
 
 var rpcTestCases = []rpctest.HarnessTestCase{
-	testGetBestBlock,
-	testGetBlockCount,
-	testGetBlockHash,
+	0: testGetBestBlock,
```

Comment:
Was this left over from your testing?  Unless I'm missing something, I don't see any benefit to this as it isn't valid to specify them sparsely.